### PR TITLE
Feat mobile fixes v1

### PR DESCRIPTION
**Pull request by kapilchothe1**
Recurring Event form , single event form .
Form UI fixes by Rishi.

### CODE REVIEW

File: components/newRecurringEvent.vue [hunk 1]
```diff
@@ -0,0 +1,2202 @@
...
+          (isInvalidSlot === false ||
+            confirm(
+              'This session overlaps with another session, are you sure? click Yes to create and cancel to cancel it.'
+            ))
+        ) {
+          this.isSaveButtonDisabled = true
+          this.eventData.EventManager = this.$auth.$state.user.data.email
+          this.eventData.Organizer = this.$auth.$state.user.data.name
+          const baseUrl = getApiUrl()
+          this.$axios
```
Comment (SukantGujar) [inline components/newRecurringEvent.vue:1962]: Use async await instead of promises.

File: components/newSingleEvent.vue [hunk 2]
```diff
@@ -561,13 +610,17 @@ export default {
     }
   },
   computed: {
+    eventLinkHint() {
+      return `https://bitpod-event.test.bitpod.io/e/${this.eventData.UniqLink}`
```
Comment (SukantGujar) [inline components/newSingleEvent.vue:614]: Hard-coded links

File: components/newSingleEvent.vue [hunk 3]
```diff
@@ -710,7 +772,62 @@ export default {
...
     },
-    next(value) {
+    validTab1() {
+      const {
+        Title,
+        StartDate,
+        EndDate,
+        Timezone,
+        UniqLink,
+        // LocationType,
```
Comment (SukantGujar) [inline components/newSingleEvent.vue:782]: No commented code.

File: components/newSingleEvent.vue [hunk 4]
```diff
@@ -710,7 +772,62 @@ export default {
...
+        // WebinarLink,
+      } = this.eventData
+      if (
```
Comment (SukantGujar) [inline components/newSingleEvent.vue:785]: Just return the condition in `if` no need for the `if-else` block.

`return (.....conditions....)`

File: nuxt.config.js [hunk 5]
```diff
@@ -106,7 +106,7 @@ export default {
   axios: {
     apiEndpoint: '/svc/api/',
     backendBaseUrl: process.env.PUBLIC_DOMAIN || '',
-    baseURL: `https://${process.env.PUBLIC_DOMAIN}${basePath}`,
+    // baseURL: `https://${process.env.PUBLIC_DOMAIN}${basePath}`,
```
Comment (SukantGujar) [inline nuxt.config.js:109]: Remove

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: components/newRecurringEvent.vue (hunk 1 above)
Comment (kapilchothe1) [reply components/newRecurringEvent.vue]: Resolved

File: components/newSingleEvent.vue (hunk 2 above)
Comment (kapilchothe1) [reply components/newSingleEvent.vue]: Resolved

File: components/newSingleEvent.vue (hunk 3 above)
Comment (kapilchothe1) [reply components/newSingleEvent.vue]: Resolved

File: components/newSingleEvent.vue (hunk 4 above)
Comment (kapilchothe1) [reply components/newSingleEvent.vue]: Resolved

File: nuxt.config.js (hunk 5 above)
Comment (kapilchothe1) [reply nuxt.config.js]: Resolved

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**